fence/README.md

```diff
@@ -0,0 +1,155 @@
+# Network Fencing
+
+## Terminology
+
+| Term     | Definition                                                                            |
+| -------- | ------------------------------------------------------------------------------------- |
+| CO       | Container Orchestration system that communicates with plugins using CSI service RPCs. |
+| SP       | Storage Provider, the vendor of a CSI plugin implementation.                          |
+| DR       | Disaster Recovery.                                                                    |
+| RPC      | [Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call).         |
+
+## Objective
+
+Define a standard that will enable storage vendors (SP) to
+perform node level fencing using corresponding CIDR blocks.
+
+### Goals in MVP
+
+The new extension will define a procedure that allows SP to blocklist
```

Comment:
blocklist is one way, and very Ceph specific. Can you add something mentioning firewalls?

Reply:
Updated the description. Let me know if there is anything specific that I am missing here. Thanks!